File: examples-sdk/csharp/802-trigger-fs-watcher/Main.cs
// 802-trigger-fs-watcher — C# SDK equivalent
// Compile: vil compile --from csharp --input 802-trigger-fs-watcher/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/java/802-trigger-fs-watcher/Main.java
// 802-trigger-fs-watcher — Java SDK equivalent
// Compile: vil compile --from java --input 802-trigger-fs-watcher/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}
